site/create.md

```diff
@@ -30,3 +30,24 @@ If the model or variables file contains encrypted passwords, add the `-use_encry
 ### Using Multiple Models
 
 The Create Domain Tool supports the use of multiple models, as described in [Using Multiple Models](../README.md#using-multiple-models).
+
+### Development Domain and "boot.properties"
```

Comment:
Development Domain and "boot.properties" -> Development domain and `boot.properties` file  (code font for `boot.properties`, not quotation marks)